Python/Video to PDF(CLI)/video_to_pdf.py

```diff
@@ -0,0 +1,78 @@
+from moviepy.editor import VideoFileClip
+from moviepy.editor import AudioFileClip
+from fpdf import FPDF
+import threading
+import speech_recognition as sr
+import os
+
+#variables
+video_clip = ''
+audio_clip = ''
+
+def audio_to_pdf():
+    '''
+    Function to convert audio to pdf:
+    Here, we first extract the audio from the selected video and store it into
+    'my_audio.wav'. Then, using the speech recognition module and Google speech API,
+    we convert this audio to text and write it into a file named 'my_text.txt'.
+    This text file is then passed to text_to_pdf function to convert it into a pdf and at
+    the end the audio and text files are removed using the OS module as we don't need them anymore.
+    '''
+    global audio_clip
+    try :
+        #extract audio
+        audio_clip = video_clip.audio.write_audiofile(r"my_audio.wav")
+        r = sr.Recognizer()
+        #open the audio file
+        with sr.AudioFile("my_audio.wav") as source:
+            #listen for the data
+            audio_data = r.record(source)
+            #recognize
+            text = r.recognize_google(audio_data)
+            #write into a file
+            write_file = open('my_text.txt', 'w')
+            write_file.write(text)
+            write_file.close()
+            #convert to pdf
+            text_to_pdf('my_text.txt')
+        print("Conversion Successfull!")
+    except:
+        print("Conversion not performed")
+
+    #delete audio and text files
+    os.remove("my_text.txt")
+    os.remove("my_audio.wav")
+
+
+def text_to_pdf(file):
+    '''
+    Function to convert text to pdf:
```

Comment:
This is not what a DocString actually is though.

Take this for [example](https://www.programiz.com/python-programming/docstrings). I am not going to nitpick here, so don't change it, for now, would be better if you keep this in mind in the future though.